server/sonar-server/src/main/java/org/sonar/server/issue/notification/AbstractNewIssuesEmailTemplate.java

```diff
@@ -88,16 +90,16 @@ public EmailMessage format(Notification notification) {
 
     return new EmailMessage()
       .setMessageId(notification.getType() + "/" + notification.getFieldValue(FIELD_PROJECT_KEY))
-      .setSubject(subject(notification, projectName))
+      .setSubject(subject(notification, projectName, branchName))
       .setMessage(message.toString());
   }
 
   protected abstract boolean shouldNotFormat(Notification notification);
 
-  protected String subject(Notification notification, String projectName) {
+  protected String subject(Notification notification, String projectName, @Nullable String branchName) {
     return String.format("%s: %s new issues (new debt: %s)",
-      projectName,
-      notification.getFieldValue(Metric.SEVERITY + COUNT),
+      branchName == null || branchName.isEmpty() ? projectName : projectName + " (" + branchName + ")",
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add parentheses to make the operator precedence explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS864)